x-pack/test/functional/apps/index_management/index_details_page.ts

```diff
@@ -18,14 +18,14 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       await pageObjects.common.navigateToApp('indexManagement');
     });
 
-    it('Navigates to the index details page from the home page', async () => {
+    it.skip('Navigates to the index details page from the home page', async () => {
```

Comment:
Did this break because of the changes or is it still wip? It seems like an important test to have

Reply:
It constantly breaks because it can't load indexManagement page and supposed to be fixed in future tickets

<img width="758" alt="image" src="https://github.com/user-attachments/assets/3128bf59-1533-4389-ac94-69a06331d070" />

Reply:
@saarikabhasi 's Index Details locator PR will fix this, and it _may_ merge first. but either way lets make sure this doesn't stay skipped.